kale/utils/metadata_utils.py

```diff
@@ -53,10 +53,11 @@ def validate_metadata(notebook_metadata):
     metadata.update(notebook_metadata)
 
     if not re.match(kale_step_name_regex, metadata['pipeline_name']):
-        raise ValueError("Pipeline name  %s" % kale_name_msg)
+        raise ValueError("Pipeline name  {}".format(kale_name_msg))
```

Comment:
This was Python3.5 compatible already :P 
But we can keep it for uniformity. (No need to comment on every occurrence)